addons/web/static/src/js/tools/test_menus.js

```diff
@@ -100,8 +99,10 @@
             return testMenuPromise;
         }).then(function(){
                 // no effect in community
-                var $homeMenu = $("nav.o_main_navbar > a.o_menu_toggle.fa-th");
-                $homeMenu.click();
+                setTimeout(function(){
+                  var $homeMenu = $("nav.o_main_navbar > a.o_menu_toggle.fa-th");
+                  $homeMenu.click();
+                }, 0);
```

Comment:
Does this really work?? This is not what we did earlier.

Reply:
Yes it works.
The previous thing was to wait for ".o_home_menu_background" with an interval

Reply:
I'm afraid it wasn't. We clicked directly on the button, and then made a Promise that resolved when the home menu was in the DOM, and we returned that promise, making the whole thing wait for it to be displayed before clicking on the next app.

Reply:
If this works, it's by chance, and we are still clicking on something that is not in the DOM.

Reply:
Ok, I revert with the previous code then.

Reply:
```
return new Promise(function (resolve) {
    setTimeout(resolve, 0);
});
```

Reply:
Hmm, I see, I thought that searching for the home menu in the setTimeout  would be sufficient to have it in the DOM